src/common/xo/index.js

```diff
@@ -841,6 +841,18 @@ export const stopVms = (vms, force = false) =>
 
 export const suspendVm = vm => _call('vm.suspend', { id: resolveId(vm) })
 
+export const suspendVms = vms =>
+  confirm({
+    title: _('suspendVmsModalTitle', { vms: vms.length }),
+    body: _('suspendVmsModalMessage', { vms: vms.length }),
+  }).then(
+    () =>
+      Promise.all(
+        map(vms, vmId => _call('vm.suspend', { id: resolveId(vmId) }))
```

Comment:
`vmId` → `vm` because it's not a VM identifier but a VM object or you don't need `resolveId`.